_opensearch/remote.md

```diff
@@ -132,4 +136,21 @@ curl -X POST "https://localhost:9200/_remotestore/_restore" -ku admin:admin -H '
 ```
 
 If the security plugin is enabled, a user must have the `cluster:admin/remotestore/restore` permission. See [Access control](/security-plugin/access-control/index/) for information about configuring user permissions.
-{: .note}
\ No newline at end of file
+{: .note}
+
+## Potential use cases
+
+The following are potential use cases for the remote-backed storage feature:
+
+- The ability to restore red clusters / indices.
+- The ability to recover all data until the last acknowledged write regardless of replica count if `index.translog.durability=request`.
+
+## Known limitations
+
+The following are known limitations of the remote-backed storage feature:
+
+- Writing data to a remote store can be a high latency operation when compared to writing data on local file system. This, in expectation, impacts the indexing throughput and latency.
```

Comment:
```suggestion
- Writing data to a remote store can be a high latency operation when compared to writing data on local file system. This may impact the indexing throughput and latency.
```